src/FSharpVSPowerTools.Logic/ResolveUnopenedNamespaceTagger.fs

```diff
@@ -14,10 +14,18 @@ open FSharpVSPowerTools.ProjectSystem
 open Microsoft.FSharp.Compiler
 open System.Threading
 
+[<NoComparison; NoEquality>]
+type Suggestion =
```

Comment:
Can we change this record type to an interface and use object expressions for instantiation? I think it's recommended in F# Component Design guidelines.